sdk/Lusid.Sdk.Tests/tutorials/Properties/TransactionProperties.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using Lusid.Sdk.Api;
+using Lusid.Sdk.Client;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Properties
+{
+    [TestFixture]
+    public class TransactionProperties : TutorialBase
+    {
+        private IList<string> _instrumentIds;
+        
+        [OneTimeSetUp]
+        public void OnetimeSetUp()
+        {
+            var instrumentLoader = new InstrumentLoader(_apiFactory);
+            _instrumentIds = instrumentLoader.LoadInstruments();
+        }
+
+        internal string _portfolioCode;
+        [SetUp]
+        public void SetUp()
+        {
+            var portfolioRequest = TestDataUtilities.BuildTransactionPortfolioRequest();
+            var portfolio = _transactionPortfoliosApi.CreatePortfolio(TestDataUtilities.TutorialScope, portfolioRequest);
+            _portfolioCode = portfolioRequest.Code;
+            Assert.That(portfolio?.Id.Code, Is.EqualTo(_portfolioCode));
+        }
+
+        [LusidFeature("F13-4")]
+        [Test]
+        public void Test_Transaction_Property()
+        {
+            var effectiveDate = new DateTimeOffset(2018, 1, 1, 0, 0, 0, TimeSpan.Zero);
+
+            //create the transaction request
+            string code = "TransactionTestDetail";
+            string perpetualPropertyKey = $"Transaction/{TestDataUtilities.TutorialScope}/{code}";
+            string perpetualPropertyPropVal = "TestPropertyLabelValue";
+
+            EnsurePropertyDefinition(code);
+
+            var transaction = new TransactionRequest(
+                // unique transaction id
+                transactionId: Guid.NewGuid().ToString(),
+
+                // instruments must already exist in LUSID and have a valid LUSID instrument id
```

Comment:
this isn't actually the case, we do allow for adding transactions without the instruments existing (see https://support.lusid.com/knowledgebase/article/KA-01860/en-us)

for now remove the comment

Reply:
Comment removed.